hooks: qt6: collect multimedia plugins for QtMultimedia >= 6.4.0

rokm opened the pull request:
In Qt6 6.4.0, QtMultimedia replaced `video/gstvideorenderer` and `video/videonode` plugins with `multimedia` plugins. Update the module information to ensure that the plugins are collected.

Fixes #7352.

rokm committed:
hooks: qt6: collect multimedia plugins for QtMultimedia >= 6.4.0

In Qt6 6.4.0, QtMultimedia replaced `video/gstvideorenderer` and
`video/videonode` plugins with `multimedia` plugins. Update the
module information to ensure that the plugins are collected.

bwoodsend commented:
I'm guessing that the QtMultimedia permutation of the tests added in c1e6bf17882c2f971ea4b90dd4619c5cf5a6fd1f just doesn't run enough code to have caught this?

rokm commented:
> I'm guessing that the QtMultimedia permutation of the tests added in [c1e6bf1](https://github.com/pyinstaller/pyinstaller/commit/c1e6bf17882c2f971ea4b90dd4619c5cf5a6fd1f) just doesn't run enough code to have caught this?

Yeah, basic import test is insufficient for catching issues like that. We can make a test out of the reproducer code in #7352, though.

rokm committed:
tests: add a test for QtMultimedia to ensure we collect plugins

Use an additional QtMultimedia test that rises erorr when the
module's plugins are missing.

rokm committed:
ci: ubuntu: install gstreamer1.0-plugins-base package

Install gstreamer1.0-plugins-base package to install base gstreamer
plugins, instead of installing just the common shared libraries
for plugins (libgstreamer-plugins-base1.0-0).
